app/templates/initiative.html

```diff
@@ -49,7 +50,7 @@ <h2>{{ initiative.name }}</h2>
             <div class="tile__container">
               <p class="tile__title">Tags:
                 {% for tag in initiative.tags.all %}
-                  <span class="tag white" style="background-color: {{ tag.color }}">{{ tag.name }}</span>
+                  <a href="{% url 'index' %}?type={{ initiative|class_name|lower }}&tag={{ tag.name }}" class="tag white" style="background-color: {{ tag.color }}">{{ tag.name }}</a>
```

Comment:
```suggestion
                  <a href="{% url 'index' %}?type={{ initiative|class_name }}&tag={{ tag.name }}" class="tag white" style="background-color: {{ tag.color }}">{{ tag.name }}</a>
```
Please check, but I'm pretty sure upper/lower case doesn't matter.

Reply:
GET parameters here seem to be case sensitive.